internal/server/gen/mocks/waypoint_server.go

```diff
@@ -1,4 +1,4 @@
-// Code generated by mockery v1.1.2. DO NOT EDIT.
+// Code generated by mockery 2.7.4. DO NOT EDIT.
```

Comment:
Could you put the mockery version change in a different PR, or maybe at least a different commit? There are a couple of places this is pinned, so it would be nice to capture the whole change at once.

Reply:
Can disregard this comment in favor of Mitchell's review :)

Reply:
You're right Rae, I will remove this from the PR since it's out of scope